includes/tools.php

```diff
@@ -369,7 +351,6 @@ function get_support_details() {
 	$report .= 'Requirements:' . PHP_EOL;
 	$report .= '- PHP >= 5.3         ' . ( $compatibility['details']['php'] ? 'PASS' : 'FAIL' ) . PHP_EOL;
 	$report .= '- HTTPS support      ' . ( $compatibility['details']['https'] ? 'PASS' : 'FAIL' ) . PHP_EOL;
```

Comment:
Not especially, since this report is written more for the plugin developers (e.g. you, @bswatson, and I) than the user.

That being said, when the support details get abstracted to a separate library making them translatable will become a priority.